### PR TITLE
[Experimental] Rating Filter: clean code about dropdown mode

### PR DESCRIPTION
**Pull request by retrofox**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR is a follow-up of https://github.com/woocommerce/woocommerce/pull/52152.
We decided to don't implement the `Dropdown` mode for this block, at least for this cycle.
This is a janitorial PR removing the code about the Dropdown representation of the block, and this, the multiple selection.

Closes # .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

0. Ensure the experimental blocks are enabled:
    - If you use the WooCommerce Beta Tester plugin, you can enable the experimental blocks in Tools > WCA Test Helper > Features. Toggle the `experimental-blocks` option.
    - If you build this PR locally, build the branch with `pnpm --filter='@woocommerce/plugin-woocommerce' build` command to have experimental blocks available.
1. Add the **Product Filters (Experimental)** block to the Product Catalog template.
2. Open the block sidebar
3. Confirm it doesn't show the Display section anymore   

Before | After
------|-----
image | image

5. Confirm the block keeps working as expected 
  -  Save and go to the shop page.
  - Select some filters, see filter blocks react, and update the filter options accordingly.
  - Interact with the **Rating** filter slider. See the filter updated accordingly.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/rating-filter/edit.tsx [hunk 1]
```diff
@@ -26,7 +26,6 @@ import type { BlockEditProps } from '@wordpress/blocks';
 import { previewOptions } from './preview';
 import { getActiveFilters } from './utils';
 import { Inspector } from './components/inspector';
-import { PreviewDropdown } from '../components/preview-dropdown';
```
Comment (dinhtungdu) [inline plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/rating-filter/edit.tsx:29]: Let's also remove the dropdown component as it's unused now.

**Review by dinhtungdu — CHANGES_REQUESTED** [review]

File: plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/rating-filter/edit.tsx (hunk 1 above)
Comment (retrofox) [reply plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/rating-filter/edit.tsx:29]: thanks! removing

Comment (retrofox) [reply plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/rating-filter/edit.tsx:29]: addressed 666055d68a260ae5c63219fa2cd8873068800677

**Review by dinhtungdu — APPROVED** [review]
Thanks for addressing the feedback!

**Pull request merged**